Bottom: 37a843b045d17d5316961aa1c771df27c7cf3ccf
Top:    93467b8bc33b83a313d9f05e2f16c8bb8f89f00f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-27 10:46:17 -0500

Add Changepassword.php page to phpldapadmin



---
diff --git a/docker/docker-phpldapadmin/Dockerfile b/docker/docker-phpldapadmin/Dockerfile
index 4544e647d..249051d65 100644
--- a/docker/docker-phpldapadmin/Dockerfile
+++ b/docker/docker-phpldapadmin/Dockerfile
@@ -1,3 +1,3 @@
 FROM osixia/phpldapadmin:latest
 
-COPY changepassword.php /var/www/changepassword.php
+COPY changepassword.php /var/www/phpldapadmin_bootstrap/htdocs/changepassword.php
